dotcom-rendering/src/experiments/tests/sign-in-gate-time-of-day.ts

```diff
@@ -0,0 +1,23 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateTimesOfDay: ABTest = {
+	id: 'SignInGateTimesOfDay',
+	start: '2023-12-04',
+	expiry: '2023-12-09',
```

Comment:
This is due to expire on Friday. We will need to sure we have a PR ready to bump the audience segmentation back to 90% on the main

Reply:
Fyi the audience segmentation changes to the main gate audience needs to be changed in both DCR and Frontend test definitions in both repos otherwise we have overlapping tests and the main variant takes precedent.